app/helpers/hiring_staff/schools_helper.rb

```diff
@@ -0,0 +1,23 @@
+module HiringStaff::SchoolsHelper
+  def table_header_sort_by(title, column:, sort:)
+    if column == sort.column
```

Comment:
Might be clearer to do this as a couple of oneliners:

```
order = (column == sort.column) ? sort.reverse_order : sort.order
active_class = (order == sort.reverse_order) ? ' active' : ''
```

That way you can remove some of the complexity in the class string interpolation below too

Reply:
Refactored this, so hopefully it reads more easily.